### ISSUE TITLE
Permission denied error mid-way through a processing sequence

### ISSUE THREAD

**Issue by tpering** [twVolc/PyCamPermanent#171]
As below, this may or may not be linked to #168  

![image](https://github.com/twVolc/PyCamPermanent/assets/7857899/4acf2a96-a9b9-475f-b785-c413278c8111)

**Comment by tpering** [comment]
Seems to happen frequently, note that this is when connected to a vpn and accessing a specific drive:

![image](https://github.com/twVolc/PyCamPermanent/assets/7857899/31902531-eadd-4907-8e5c-545cf24a2503)

**Comment by tpering** [comment]
A further note, this then prevents the saving of the updated .csv which means that if processing of images continues (as seems to be the case), then it results in a "No DOAS data point within 2s of image time" message.

**Comment by ubdbra001** [comment]
This looks different from #168 (it looks like permission problems loading the data rather than saving it), have you encountered it since you initially opened the issue?

**Comment by tpering** [comment]
Yes, but this only occurs when trying to use the "Start Watching" directory option of RTP. It also occurs when connected to the X drive and data is being transferred into there. I suspect something is happening with the code trying to grab images before they are fully loaded? This would be good to fix so we can test (fully use) RTP for our satellite telemetry conmections
